src/wagtail_localize_smartling/templatetags/wagtail_localize_smartling_admin_tags.py

```diff
@@ -22,21 +22,26 @@ def smartling_edit_translation_message(context):
     }
 
     translation = context["translation"]
-    jobs = translation.smartling_jobs.all()
+    jobs = translation.smartling_jobs.exclude(
+        status__in=[JobStatus.CANCELLED, JobStatus.DELETED]
```

Comment:
Issue: replace this with `sync.UNTRANSLATED_STATUSES` so we're consistent about which statuses are final, but didn't result in the job having translations for us to import.

Reply:
I forgot about those. Replaced